app/kubernetes-ingress-controller/1.0.x/guides/configuring-fallback-service.md

```diff
@@ -43,9 +43,53 @@ For the purpose of this guide, we will setup an [httpbin](https://httpbin.org)
 service in the cluster and proxy it.
```

Comment:
I'm confused. The text says this is an example for httpbin, but the example appears to be for fallback-svc - was this intentional?

Reply:
Probably copy-pasta from some other example that did use HTTPbin, whereas this uses `image: hashicorp/http-echo`.

b5e95d5 uses generic language to describe the application instead. "fallback" here more describes what we're using it for (a service that handles requests that don't match a rule for something else), and the actual application used isn't particularly important.